collision_detection_fcl/src/collision_robot_fcl.cpp

```diff
@@ -72,16 +84,19 @@ void collision_detection::CollisionRobotFCL::getAttachedBodyObjects(const robot_
 void collision_detection::CollisionRobotFCL::constructFCLObject(const robot_state::RobotState &state, FCLObject &fcl_obj) const
 {
   fcl_obj.collision_objects_.reserve(geoms_.size());
+  fcl::Transform3f tf;
```

Comment:
I don't like the var name ``tf`` because it can confuse ROS users used to the ``TF`` package